### ISSUE TITLE
Compatibility issue with Django 1.10

### ISSUE THREAD

**Issue by Kianoosh76** [django-polymorphic/django-polymorphic-tree#55]
I'm giving this attribute error while upgrading Django version to 1.10
other package versions:

```
django-mptt==0.8.6
django-polymorphic==1.0
django-polymorphic-tree==1.2.5
```

When I run the server this error appears:

```
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/django/core/management/__init__.py", line 341, in execute
    django.setup()
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/django/apps/config.py", line 199, in import_models
    self.models_module = import_module(models_module_name)
  File "//anaconda/envs/lyan/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 662, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/polymorphic_tree/models.py", line 71, in <module>
    class PolymorphicMPTTModel(with_metaclass(PolymorphicMPTTModelBase, MPTTModel, PolymorphicModel)):
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/future/utils/__init__.py", line 137, in __new__
    return meta(name, bases, d)
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/mptt/models.py", line 263, in __new__
    cls = super_new(meta, class_name, bases, class_dict)
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/polymorphic/base.py", line 68, in __new__
    new_class = self.call_superclass_new_method(model_name, bases, attrs)
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/polymorphic/base.py", line 202, in call_superclass_new_method
    new_class = super(PolymorphicModelBase, self).__new__(self, model_name, bases, attrs)
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/django/db/models/base.py", line 157, in __new__
    new_class.add_to_class(obj_name, obj)
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/django/db/models/base.py", line 316, in add_to_class
    value.contribute_to_class(cls, name)
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/mptt/managers.py", line 82, in contribute_to_class
    super(TreeManager, self).contribute_to_class(model, name)
  File "//anaconda/envs/lyan/lib/python3.5/site-packages/django/db/models/manager.py", line 120, in contribute_to_class
    setattr(model, name, ManagerDescriptor(self))
AttributeError: can't set attribute
```

**Comment by vdboor** [comment]
Could you try the current git release?

```
pip uninstall django-polymorphic
pip install -e git+git@github.com:edoburu/django-polymorphic-tree.git@53abb66bed21e73a6b1775cb539801e076ccd513#egg=django_polymorphic_tree
```

There are changes in git for Django 1.10 compatibility, but these still need to be tested before 1.3 is released.

**Comment by Kianoosh76** [comment]
I tried that bot nothing changed
Looks like it does not supports django 1.10 with python 3.5

**Comment by vdboor** [comment]
I've just released 1.3, which does fix some things.
However, I noticed that Django 1.10.1 breaks polymorphic again, so you're better off at using Django 1.9.9 for now.

Typically when a new Django release is made, most third party apps are generally up to date, but some rough edges remain. For existing projects, I'd recommend using Django 1.9 for a few months until those edges are fixed.
